tft.py

```diff
@@ -446,10 +448,12 @@ def check_if_game_complete(wait_for_exit_buttons: bool = False) -> bool:
                 logger.info("The game considers us dead, checking the screen again for exit buttons")
                 return check_if_game_complete(wait_for_exit_buttons=True)
 
-            logger.warning("We are in-game and considered dead, but no death button has been found.")
-            logger.info("Clicking approximate 'Exit Now' location, please ensure your game is in English.")
-            auto.moveTo(827, 553)
-            click_left()
+            logger.warning("We are in-game and considered dead, but no death button has been found. Exiting gracefully")
+            # A non-forceful taskkill tries to tell the process to exit gracefully, similar to Alt+F4.
+            # The first request opens the "Are you sure you want to exit?" window.
+            parse_task_kill_result(kill_process(CONSTANTS["processes"]["game"], force=False))
+            # The second request "confirms" the wish.
+            parse_task_kill_result(kill_process(CONSTANTS["processes"]["game"], force=False))
             time.sleep(5)

```

Comment:
Should we add a check after the 5 second mark to see if the game executable is still running, and if so, to kill it?

Reply:
I've been considering changing this to wait for the client to report that it's no longer in a game, so this is a good idea.